develop/toolkit/parachains/quickstart/pop-cli.md

```diff
@@ -0,0 +1,84 @@
+---
+
+# Quickstart Parachain Development With Pop CLI
+
+## Introduction
+
+[Pop CLI](https://onpop.io/cli/){target=\_blank} is a powerful command-line tool designed explicitly for rapid parachain development within the Polkadot ecosystem. It addresses essential developer needs by providing streamlined commands to set up development environments, scaffold parachain templates, and manage local blockchain networks.
+
+Pop CLI simplifies parachain development with features like:
+
+- Quick initialization of parachain development environments
+
+- Project scaffolding from predefined parachain templates
+
+- Easy deployment and management of local development networks
+
+Developers can quickly begin coding and testing, significantly reducing setup overhead.
+
+### Install Pop CLI
+
+To install Pop CLI, run the following command:
+
+```bash
+cargo install --force --locked pop-cli
+```
+
+Confirm that Pop CLI is installed by running `pop --help` in your terminal:
+
+```bash
+pop --help
+```
+
+### Set Up Your Development Environment
+
+To develop and build Polkadot SDK-based it's essential to prepare your local environment with the necessary tools and dependencies. The [Install Polkadot SDK Dependencies](/develop/parachains/install-polkadot-sdk/){target=\_blank} guide walks you through this setup step-by-step.
+
+However, you can automate this entire process by running:
+
+```bash
+pop install
+```
+
+This command provides an interactive experience that checks for all necessary dependencies and installs them for you. It’s the fastest and easiest way to get your development environment ready for building parachains with Pop CLI.
+
+--8<-- 'code/develop/toolkit/parachains/quickstart/pop-cli/install.html'
+
+### Initialize a Project
+
+Start a new project quickly using Pop CLI `pop new parachain` command:
+
+--8<-- 'code/develop/toolkit/parachains/quickstart/pop-cli/new-parachain.html'
+
+The command above scaffolds a new parachain project using the default template included with Pop CLI. For more specialized implementations, additional templates are available, you can explore them by running `pop new parachain --help`.
+
+Once the project is generated, move into the new directory and build your parachain:
+
+```
+cd my-parachain
+pop build --release
+```
+
+!!! note
+    Under the hood, `pop build --release` runs `cargo build --release`, but `pop build` adds functionality specific to Polkadot SDK projects, such as [deterministic runtime builds](/develop/parachains/deployment/build-deterministic-runtime/){target=\_blank} and automatic management of feature flags like `benchmark` or `try-runtime`.
+
+Pop CLI integrates the [Zombienet-SDK](https://github.com/paritytech/zombienet-sdk){target=\_blank} allowing you to easily launch ephemeral local networks for development and testing. To start a network, simply run the following:
```

Comment:
```suggestion
Pop CLI integrates the [Zombienet SDK](https://github.com/paritytech/zombienet-sdk){target=\_blank}, allowing you to easily launch ephemeral local networks for development and testing. To start a network, simply run the following:
```

Reply:
Fixed in https://github.com/polkadot-developers/polkadot-docs/pull/538/commits/cf6091a8a444f08447db2bdaa88438254130b286